pkg/plugins/plugin_spec.go

```diff
@@ -0,0 +1,47 @@
+package plugins
+
+import (
+	"log"
+
+	"gopkg.in/yaml.v2"
+)
+
+const GithubRaws = "https://raw.githubusercontent.com/kf5i/k3ai-plugins/main/"
```

Comment:
Ideally, this path does not need to be hardcoded. It can be read from the Github api response. This would also make it easier to work with a Github enterprise plugin repository if we get there.

Reply:
yes, I put it there to test the API. I'd open another issue btw.